config/initializers/inflections.rb

```diff
@@ -2,4 +2,5 @@
 ActiveSupport::Inflector.inflections do |inflect|
   inflect.acronym 'EVSS'
   inflect.acronym 'GI'
+  inflect.uncountable 'terms_and_conditions'
```

Comment:
This is just to work around the fact that terms_and_conditions are always pluralized, right?